src/App.js

```diff
@@ -31,34 +34,44 @@ const StyledResultsContainer = styled.div`
 
 function App() {
   //adding pieces of state with the currentPrice of our currency, our Reference selection, our BetBehaviour and the NextPrice
+  const [currency, setCurrency] = useState('BTC');
   const [currentPrice, setCurrentPrice] = useState(null);
   const [referencePrice, setReferencePrice] = useState(null);
   const [betType, setBetType] = useState(null);
   const [betPrice, setBetPrice] = useState(null);
   const [showResults, setShowResults] = useState(false);
   const [message, setMessage] = useState(null);
+  const [credits, setCredits] = useState(3);
 
   // instantiating our currency formatter
   let formattedCurrency = new Intl.NumberFormat('en-US', {
     style: 'currency',
     currency: 'USD',
   }).format(currentPrice);
 
-  // function to setReferencePrice when button onClick
+  // function to setReferencePrice when Button onClick
   const selectReference = () => {
     setReferencePrice(formattedCurrency);
     console.log('reference: ' + formattedCurrency);
   };
 
+  // function to select t(he currency
+  const selectCurrency = (e) => {
+    console.log(e.target.value);
```

Comment:
I'd remove the console logs from prs. 
A fun thing you can do (not for now) is setup different environments - i.e. staging vs. production. 

You can then write utils that will only console log if you are in staging and/or if you are an admin (but admin === auth and idk if we want to go down that road right now)